base/array.jl

```diff
@@ -320,6 +320,8 @@ oneunit(x::AbstractMatrix{T}) where {T} = _one(oneunit(T), x)
 
 convert(::Type{Vector}, x::AbstractVector{T}) where {T} = convert(Vector{T}, x)
 convert(::Type{Matrix}, x::AbstractMatrix{T}) where {T} = convert(Matrix{T}, x)
+convert(::Type{Matrix}, x::Vector{T}) where {T} = Matrix(reshape(x, :, 1))
+convert(::Type{Matrix}, x::RowVector{T,S}) where {T,S} = Matrix(reshape(x, 1, :))
```

Comment:
Ah, yes, I see.  Well in the new code that I've pushed on this branch I don't use `reshape()` at all, which is probably the right thing to do, because you wouldn't want a `convert()` call to be a view into the original data anyway, I don't think.